cmd/konnect_diff.go

```diff
@@ -16,11 +16,11 @@ var (
 var konnectDiffCmd = &cobra.Command{
 	Use:   "diff",
 	Short: "Diff the current entities in Konnect with the one on disks (in alpha)",
-	Long: `Diff is like a dry run of 'decK sync' command.
+	Long: `The Konnect diff command is similar to a dry run of 'decK sync' command.
```

Comment:
```suggestion
	Long: `The konnect diff command is similar to a dry run of the 'deck konnect sync' command.
```
Should konnect diff be in single quotes? Maybe even 'deck konnect diff'?